[2020-04-26 20:06:07,190] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.consulta_dados_company 2020-04-26T20:06:00.213252+00:00 [queued]>
[2020-04-26 20:06:07,255] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.consulta_dados_company 2020-04-26T20:06:00.213252+00:00 [queued]>
[2020-04-26 20:06:07,256] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 20:06:07,265] {{taskinstance.py:867}} INFO - Starting attempt 1 of 2
[2020-04-26 20:06:07,269] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 20:06:07,335] {{taskinstance.py:887}} INFO - Executing <Task(PythonOperator): consulta_dados_company> on 2020-04-26T20:06:00.213252+00:00
[2020-04-26 20:06:07,348] {{standard_task_runner.py:53}} INFO - Started process 27490 to run task
[2020-04-26 20:06:07,597] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: ReclameAqui.consulta_dados_company 2020-04-26T20:06:00.213252+00:00 [running]> 030a07de9129.semantix
[2020-04-26 20:06:07,683] {{logging_mixin.py:112}} INFO - DEBUG GET: {'company': 'Cielo'}
[2020-04-26 20:06:07,684] {{logging_mixin.py:112}} INFO - DEBUG GET 1: http://api_persistenciadados:5005/scores/
[2020-04-26 20:06:07,685] {{logging_mixin.py:112}} INFO - DEBUG GET 2: {'company': 'Cielo'}
[2020-04-26 20:06:07,748] {{logging_mixin.py:112}} INFO - DEBUG GET 3: {"20200426CIELO": {"company": "Cielo", "date": "20200426", "answered": "6666", "reclamations": "6151", "response_time": "15 dias e 21 horas ", "unanswered": "113"}, "20190101CIELO": {"company": "Cielo", "date": "20190101", "answered": "6666", "reclamations": "6151", "response_time": "15 dias e 21 horas ", "unanswered": "113"}}
[2020-04-26 20:06:07,749] {{logging_mixin.py:112}} INFO - 200
[2020-04-26 20:06:07,754] {{logging_mixin.py:112}} INFO - {"20200426CIELO": {"company": "Cielo", "date": "20200426", "answered": "6666", "reclamations": "6151", "response_time": "15 dias e 21 horas ", "unanswered": "113"}, "20190101CIELO": {"company": "Cielo", "date": "20190101", "answered": "6666", "reclamations": "6151", "response_time": "15 dias e 21 horas ", "unanswered": "113"}}
[2020-04-26 20:06:07,755] {{python_operator.py:114}} INFO - Done. Returned value was: None
[2020-04-26 20:06:07,803] {{taskinstance.py:1048}} INFO - Marking task as SUCCESS.dag_id=ReclameAqui, task_id=consulta_dados_company, execution_date=20200426T200600, start_date=20200426T200607, end_date=20200426T200607
[2020-04-26 20:06:17,034] {{logging_mixin.py:112}} INFO - [2020-04-26 20:06:17,033] {{local_task_job.py:103}} INFO - Task exited with return code 0
